vmdk_plugin/drivers/photon/photon_driver.go

```diff
@@ -532,7 +520,7 @@ func (d *VolumeDriver) Create(r volume.Request) volume.Response {
 		return volume.Response{Err: errGetDevicePath.Error()}
 	}
 
-	errMkfs := fs.Mkfs(supportedFs[r.Options[fsTypeTag]], r.Name, device)
+	errMkfs := fs.MkfsByDevicePath(r.Options[fsTypeTag], r.Name, device)
```

Comment:
`Mkfs(..)` only handles `VolumeDevSpec`s, and as [golang doesn't support method overloading](https://golang.org/doc/faq#overloading), I thought of creating `MkfsByDevicePath(..)` to handle string based device paths.